<?xml version="1.0"?>
<launch>
  <!-- <group ns="tb3_0">
    <param name="robot_name" type="str" value="tb3_0" />
    <node name="drawaline" pkg="driving_turtlebots" type="line.py" output="screen"/>
  </group>
  <group ns="tb3_1">
    <param name="robot_name" type="str" value="tb3_1" />
    <node name="drawaline" pkg="driving_turtlebots" type="line.py" output="screen"/>
  </group>
  <group ns="tb3_2">
    <param name="robot_name" type="str" value="tb3_2" />
    <node name="drawaline" pkg="driving_turtlebots" type="line.py" output="screen"/> -->
  <!-- </group> -->
  <include file="$(find driving_turtlebots)/launch/line_tb3_0.launch"/>

  <include file="$(find driving_turtlebots)/launch/line_tb3_1.launch"/>

  <include file="$(find driving_turtlebots)/launch/line_tb3_2.launch"/>

</launch>
